### ISSUE TITLE
Remove most region locked building GFX

### ISSUE THREAD

**Issue by JukeJukestar** [CK3RealmsinExile/RealmsInExile#764]
(Copying Jauff's message from the suggestions thread)

 I wanted to point out the fact that culture groups such as the Eotheod and elves do not use their unique building gfx outside of specific regions. Now of course elves, Eotheod and Numenoreans all have correct gfx that they use. There are no unintentional things going on. However while playing as the Framlings I was rather dissapointed to see them using the standard dunlending gfx in the anduin vale for their holdings (leofrings and Rohirrim as well of course) and as Lothlorien I was dissapointed to not be able to spread the malorn tree holdings as I took Dol Guldur and spread Galathrim culture. 

What I am suggesting is that the Malorn tree gfx be tied to the galathrim culture as their building gfx for anywhere they spread. I have spoken before to people not on the dev team who were sad that it is not the case as well. I also suggest that the region lock on the Rohirrim gfx be removed, so that the Framlings can use it in the vale, and Rohan can use it if they colonize Dunland. I think that the Numenorean region lock works well for showing the difference in their building materials between the south and the north though and would find it odd to see marble Gondorian buildings popping up in a conquered Umbar, or Harondor and besides; those gfx are both used plenty. 

TLDR: I suggest some lesser used custom holding GFX be used more often

**Comment by Jauffr** [comment]
Another minor gripe I have with how this is currently set up is how any culture that takes over certain regions will not be able to change the way buildings look. If Szrel-Kain is taken by Medloshukain and culture converted. Upgrading the buildings, or holdings there will not change the designs to the easterling GFX. While Cultures like Gondorian still change the GFX. Again. Still not a big deal and definately not release critical, but it is odd and something I would rather not be the case
![Screenshot 2023-03-20 192656](https://user-images.githubusercontent.com/108106660/226498919-1ca8476f-c72d-453f-bcf8-5c22c257c82d.png)

**Comment by jj248** [comment]
Agree, but let's leave this for v3.1 at this stage.

**Comment by Jaco-Daan** [comment]
This has been dealt with, only needs to be merged into the master branch. CLosing
